cmd/flags/root.go

```diff
@@ -46,7 +45,10 @@ func (r *Root) String() string {
 
 // InitFlags initialises all the flags.
 func (r *Root) InitFlags() {
-	flag.IntVar(&r.MaxDurationSeconds, "max-duration-seconds", 60, "Max duration in seconds after which warm up will stop making requests")
+	// TODO: rename this to `max-global-duration-seconds`
+	flag.IntVar(&r.MaxDurationSeconds, "max-duration-seconds", 60, "Global maximum duration. This includes both the time spent warming up the target service and also the time waiting for the target to become ready")
+	flag.IntVar(&r.MaxReadinessWaitSeconds, "max-readiness-wait-seconds", 30, "Maximum time to wait for the target to become ready")
```

Comment:
wondering if we should have a default here, or just have nil/-1 instead

Reply:
I think having a default makes sense.

Reply:
As long as we document it a default should be ok